### PR TITLE
Jest test : Create and Edit Co-working space

### PR DESCRIPTION
**Pull request by i300m**
write the test for Create Co-working space and Edit 100% covered

### CODE REVIEW

**Comment by thepbordin** [comment]
DEPLOYMENT FAILED DONT MERGE na

**Comment by Nonnyss** [comment]
gudgud

File: Backend/coverage/coverage-final.json
Comment (Nonnyss) [inline Backend/coverage/coverage-final.json:1]: วดฟ

Comment (Nonnyss) [reply Backend/coverage/coverage-final.json:1]: oh gudๆ

**Review by Nonnyss — APPROVED** [review]
all code not going to explode main

**Pull request merged**